src/main/scala/is/hail/variant/Variant.scala

```diff
@@ -366,6 +367,51 @@ case class Variant(contig: String,
     0
   }
 
+  def unifyAltAlleles(other: Variant): Tuple2[Variant, Variant] = {
```

Comment:
I also think this doesn't belong as a method on Variant. How about a static method on an `AnnotateAllelesVDS` object?